### ISSUE TITLE
Run without root

### ISSUE THREAD

**Issue by razzeee** [lawl/NoiseTorch#201]
Reading back the issues in this repo (e.g. https://github.com/lawl/NoiseTorch/issues/33#issuecomment-662566283) and the code, it seems like it's needed to get around something in pulse. So would it be needed, with the pipewire codepath?

It also seems to be used for the update feature, which isn't a priority for me.

**Comment by lawl** [comment]
> So would it be needed, with the pipewire codepath?

No. But my philosophy here is that a user shouldn't need to know if they run PA or PW. Like, on Windows for example, you would not expect a user to need to know if an application uses WASAPI or DirectSound.

Not requiring it for PipeWire would just make things *more* complicated, because now the install instructions depend on what audio server you use. That's why I haven't done this. (It would be very simple, just a few `if's`).

> It also seems to be used for the update feature, which isn't a priority for me.

No, the update feature just needs to set the permissions again, because linux removes them when we change the binary.

My current thinking is that PA support will probably dropped/be deprecated, half a year to a year after most distros ship it by default.

**Comment by razzeee** [comment]
Your obviously right, with that mindset.

Would it be fine for you, to let this issue be open, until we can drop PA?
That way I (or anyone else interested) will be updated, I could then try to make this work with as a flatpak again.

**Comment by lawl** [comment]
> Would it be fine for you, to let this issue be open, until we can drop PA?

sure. let's do that.

**Comment by TomyLobo** [comment]
I'm running noisetorch as a --user systemd service, i.e. without root on Kubuntu 18.04, which uses PulseAudio 11.1.
